frontend/src/reusables/urls.js

```diff
@@ -0,0 +1,3 @@
+const BASE_URL = 'https://project-auth-mg.herokuapp.com/'
+
+export const API_URL = (slug) => `${BASE_URL}/${slug}`
```

Comment:
data from my devtools: `Request URL: https://project-auth-mg.herokuapp.com//signup` you have `//` because your `BASE_URL` is ending with `/` and you are adding one more here.